contrib/cirrus/runner.sh

```diff
@@ -22,62 +22,83 @@ OS_RELEASE_ID="$(source /etc/os-release; echo $ID)"
 OS_RELEASE_VER="$(source /etc/os-release; echo $VERSION_ID | tr -d '.')"
 # Combined to ease some usage
 OS_REL_VER="${OS_RELEASE_ID}-${OS_RELEASE_VER}"
+# This is the magic interpreted by the tests to allow modifying local config/services.
+export SKOPEO_CONTAINER_TESTS=1
 
 export "PATH=$PATH:$GOPATH/bin"
 
-podmanmake() {
-    req_env_vars GOPATH SKOPEO_PATH SKOPEO_CI_CONTAINER_FQIN
-    warn "Accumulated technical-debt requires execution inside a --privileged container.  This is very likely hiding bugs!"
-    showrun podman run -it --rm --privileged \
-        -e GOPATH=$GOPATH \
-        -v $GOPATH:$GOPATH:Z \
-        -w $SKOPEO_PATH \
-        $SKOPEO_CI_CONTAINER_FQIN \
-            make "$@"
-}
-
 _run_setup() {
+    local mnt
+    local errmsg
```

Comment:
These variables aren’t set here AFAICS (but exist in `_run_integration`).

Reply:
Yep, I moved 'em.  You'll see, it's fixed in my next push (waiting for images to build first).